src/Catalyst.Cli.Tests/UnitTests/Commands/Response/GetFileFromDfsResponseTests.cs

```diff
@@ -42,7 +42,7 @@ public void GetFileFromDfsResponse_Can_Get_Output()
             //Arrange
             var getFileFromDfsResponse = new GetFileFromDfsResponse();
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
-            var getFileFromDfsCommand = new GetFileCommand(null, commandContext, Substitute.For<ILogger>());
+            new GetFileCommand(null, commandContext, Substitute.For<ILogger>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'GetFileCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)